l3kernel/l3basics.dtx

```diff
@@ -1692,6 +1692,7 @@
 %       \use_v:nnnnnnnnn , \use_vi:nnnnnnnnn , \use_vii:nnnnnnnnn , \use_viii:nnnnnnnnn ,
 %       \use_ix:nnnnnnnnn
 %   }
+% \begin{macro}[EXP]{\use_i_ii:nnn}
```

Comment:
Maybe also move its definition to the back, between `\use_<n>:n...` and `\use_ii_i:nn`, so implementation and documentation are more aligned.